src/include/distributed/connection_management.h

```diff
@@ -47,7 +47,7 @@ enum MultiConnectionMode
 	FOR_DML = 1 << 2,
 
 	/* open a connection per (co-located set of) placement(s) */
```

Comment:
can we make this comment more precise?

`/* connection must not have accessed any non-co-located placements */`